modules/oe_translation_remote/src/Form/RemoteTranslationNewForm.php

```diff
@@ -0,0 +1,258 @@
+   */
+  public function translatorConfigurationAjaxCallback(array $form, FormStateInterface $form_state): array {
+    return $form['translator_configuration'];
+  }
+
+  /**
+   * Process callback for the translator configuration form.
+   *
+   * @param array $element
+   *   The form element.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   *
+   * @return array
+   *   The translator configuration form.
+   */
+  public static function processTranslatorConfiguration(array &$element, FormStateInterface $form_state): array {
+    /** @var \Drupal\oe_translation_remote\RemoteTranslationProviderInterface $plugin */
+    $plugin = $element['#plugin'];
+    $subform_state = SubformState::createForSubform($element, $form_state->getCompleteForm(), $form_state);
+
+    return $plugin->newTranslationRequestForm($element, $subform_state);
+  }
+
+  /**
+   * Saves the new translation request and directly sends it.
+   *
+   * @param array $form
+   *   The form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   */
+  public function saveAndSend(array &$form, FormStateInterface $form_state): void {
+    $translator_id = $form_state->getValue('translator');
+    $element = $form['translator_configuration'][$translator_id];
+    $plugin = $element['#plugin'];
+    $subform_state = SubformState::createForSubform($element, $form_state->getCompleteForm(), $form_state);
+    $subform_state->set('translator_id', $translator_id);
+
+    $plugin->submitRequestToProvider($subform_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    // Stays empty.
+  }
+
+  /**
+   * Builds the existing requests form.
+   *
+   * @param array $form
+   *   The form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The current entity.
+   * @param \Drupal\oe_translation_remote\TranslationRequestRemoteInterface[] $requests
+   *   The existing requests.
+   */
+  protected function buildExistingRequestsForm(array $form, FormStateInterface $form_state, ContentEntityInterface $entity, array $requests): array {
+    // If we only have a single request, render it directly here.
+    if (count($requests) === 1) {
+      $request = reset($requests);
+
+      $form['title'] = [
+        '#type' => 'inline_template',
+        '#template' => "<h3>{{ 'Ongoing remote translation request via '|t }} {{ translator }} </h3>",
+        '#context' => [
+          'translator' => $request->getTranslatorProvider()->label(),
+        ],
+      ];
+
+      $view_builder = $this->entityTypeManager->getViewBuilder('oe_translation_request');
+      $build = $view_builder->view($request);
+      $form['request'] = $build;
+
+      return $form;
+    }
+
+    // Otherwise, create a table of requests that link to their pages.
```

Comment:
This should not happen, it's just a bit of future handling if we ever have multiple providers that can make parallel requests.